src/functions/transactor.ts

```diff
@@ -26,14 +48,18 @@ export type TTransactor = (
  * @param provider
  * @param gasPrice
  * @param etherscan
- * @returns (transactor) a function to transact which calls a callback method parameter on completion
+ * @param throwOnError - throwOnError default value its false, if true it will throw errors.
+ * @param filterErrorMessage - receive the @link TxErrorType to custom your errors message and description at notification.

```

Comment:
I need fix the @link here cuz i changed the name. But will wait for your review in case need change something else before.